Source/WebCore/dom/Element.cpp

```diff
@@ -2223,6 +2223,12 @@ void Element::notifyAttributeChanged(const QualifiedName& name, const AtomString
 
 void Element::attributeChanged(const QualifiedName& name, const AtomString& oldValue, const AtomString& newValue, AttributeModificationReason reason)
 {
+    Ref document = this->document();
+    if (isElementReflectionAttribute(document->settings(), name) || isElementsArrayReflectionAttribute(name)) {
```

Comment:
Ugh... we should flip the order of evaluation in isElementReflectionAttribute. We should check name == HTMLNames::popovertargetAttr and name == HTMLNames::commandforAttr before checking settings.popoverAttributeEnabled() or settings.invokerAttributesEnabled().